docs/content/configuration/csi_ug_config_create_storageclasses_topology.md

```diff
@@ -0,0 +1,45 @@
+# Creating a StorageClass with topology feature
+
+When using the CSI Topology feature, different parameters must be taken into account when creating a storage class yaml file with specific `by_system_id` requirements. Use this information to help define a StorageClass that is topology aware.
+
+**Note:** For information and parameter definitions that are not related to topology awareness, be sure to see the information provided in [Creating a StorageClass](csi_ug_config_create_storageclasses.md), in addition to the current section.
+
+
+The StorageClass file must be defined to contain topology information, based off of the labels that were already defined on the nodes in the cluster (see [Compatibility and requirements](content\installation\csi_ug_requirements.md)). This determines the storage pools that are then served as candidates for PersistentVolumeClaim (PVC) requests made, as well as the subset of nodes that can make use of the volumes provisioned by the CSI driver.
+
+With topology awareness, the StorageClass must have the `volumeBindingMode` set to `WaitForFirstConsumer` (as defined in the yaml example below.) This defines that any PVCs that are requested with this specific StorageClass, will wait to be configured until the CSI driver can see the worker node topology.
+
+The `by_system_id` parameter is optional and values, such as the `pool`, `SpaceEfficiency`, and `volume_name_prefix` may all be specified:
+- Within by_system_id, per system
+- Outside of the parameter, as a cross-system default
+- Both inside and outside of the parameter, as specified above
```

Comment:
`Both` can be confusing, if it is "both" - how the user can know what will be chosen for the volume?

Reply:
@oriyarde please advise. This was based off of information that you gave me. we can reassess the wording.

Reply:
I agree that when specifying both, it is important for us to clarify in the docs what will be chosen for the volume.
but I don't see an issue with the wording of `Both` by itself (assuming additional clarification is added)

Reply:
@oriyarde  
"it is important for us to clarify in the docs what will be chosen for the volume."
In this case if both are specified- which one will be chosen for the volume? I can clarify.

Reply:
the final parameter value is chosen according to this hierarchy:
1. within `by_system_id` (if the parameter is specified under the relevant system of the volume)
2. outside `by_system_id`
3. default parameter value

Reply:
perhaps this hierarchy isn't clear, let me clarify - if a parameter is specified both in `by_system_id` and outside, the chosen value will be of that within `by_system_id`, because `by_system_id` is at the top of the hierarchy.
`3.` will be chosen if the parameter was not specified neither within nor outside `by_system_id`

Reply:
updated per my conversation with @ArbelNathan 
DONE